packages/migration/src/migrate.js

```diff
@@ -129,4 +129,21 @@ export function migrate(storyData, version) {
   return result;
 }
 
+export function migrateSingle(storyData, version) {
```

Comment:
<!-- propose improvements -->
**💬 suggestion:**

Would be nice to have a test for this, too.

Reply:
Currently actually pondering how useful this new util is.